apps/coscrad-frontend/src/emotion-components/buttons.ts

```diff
@@ -0,0 +1,12 @@
+import styled from '@emotion/styled';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import { primaryColor, secondaryColor } from './colors';
+
+export const BackArrowIcon = styled(ArrowBackIosIcon)`
+    color: ${primaryColor[100]};
```

Comment:
Presumably you would inject the 100 form a config? Maybe theme.colourPalletId or something?